composer.json

```
@@ -1,7 +1,7 @@
 {
     "require": {
         "php": ">=7.4",
-        "phpstan/phpstan": "^1.8",
+        "phpstan/phpstan": "1.8.x-dev",
```

requires next phpstan release

```suggestion
        "phpstan/phpstan": "^1.8.7",
```